adcc/backends/testing.py

```diff
@@ -189,21 +189,33 @@ def cached_backend_hf(backend, molecule, basis, multiplicity=1, conv_tol=1e-12,
 
     global __cache_cached_backend_hf
 
+    # so you can only assign one of them.
+    pe_options = None
+    pcm = None
+    if env[0] == "pcm":
+        pcm = env[1]
+    elif env[0] == "pe":
+        pe_options = env[1]
```

Comment:
This could also be much simpler if we can generalize the environment keyword passing somehow....

Reply:
Agree. This looks horrible.